app/models/search/query_result.rb

```diff
@@ -82,7 +82,7 @@ def facets
     if @facets.nil?
       @facets = {}
       response['aggregations'].each_pair do |term, results|
-        if Tag::TYPES.include?(term.classify) || term == 'tag'
+        if Tag::TYPES.include?(term.classify) || term == 'tag' || term == 'archive_warning'
```

Comment:
Is this change necessary? `"archive_warning".classify` is `"ArchiveWarning"`, which (I believe) should be included in `Tag::TYPES`.

Reply:
Agreed. Not necessary.